index.html

```diff
@@ -2,11 +2,14 @@
 <html>
   <head>
     <title>Super Fly Bowling</title>
-    <script src="js/jquery.js"></script>
-    <script src="js/lodash.js"></script>
+    <script src="js/vendor/jquery.js"></script>
+    <script src="js/vendor/lodash.js"></script>
+    <script src="js/main.js"></script>
     <link rel="stylesheet" href="css/styles.css" />
   </head>
   <body>
     <div id="score">The score</div>
+    <div id="totalscore">Total score</div>
```

Comment:
Maybe it would be good for it to say 
'Score: <score>' 
'Total score: <totalScore>'